SimpleNotePad.java

```diff
@@ -0,0 +1,83 @@
+
+package simplenotepad;
+
+import java.awt.Dimension;
+import java.util.ArrayList;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+import javax.swing.JScrollPane;
+import javax.swing.JTextPane;
+
+public class SimpleNotePad extends JFrame{
+	//renamed variables lines 29-38 for increased readability
+    JMenuBar menuBar = new JMenuBar();
+    JMenu fileMenu = new JMenu("File");
+    JMenu editMenu = new JMenu("Edit");
+	 JMenu openRecent = new JMenu("Open Recent");
+    JTextPane pane = new JTextPane();
+    JMenuItem newFile = new JMenuItem("New File");
+    JMenuItem saveFile = new JMenuItem("Save File");
+    JMenuItem printFile = new JMenuItem("Print File");
+	 JMenuItem openFile = new JMenuItem("Open File");
+	//delete undo as it is not implemented
+    JMenuItem copy = new JMenuItem("Copy");
+    JMenuItem paste = new JMenuItem("Paste");
+	 JMenuItem replace = new JMenuItem("Replace");
+	 static ArrayList<String> recentFiles = new ArrayList<String>(5);
+	 
+    public SimpleNotePad() {
+        setTitle("A Simple Notepad Tool");
+        fileMenu.add(newFile);
+        fileMenu.addSeparator();
+        fileMenu.add(saveFile);
+        fileMenu.addSeparator();
+        fileMenu.add(printFile);
+		  fileMenu.addSeparator();
+		  fileMenu.add(openFile);
+		  addRecent();
+		  fileMenu.add(openRecent);
+        editMenu.add(undo);

```

Comment:
great job for taking out all the implementation about undo
but you forgot to take some undo in the simple note pad
so just go along your code and clear all the undo
implementation.